vb.go

```diff
@@ -0,0 +1,54 @@
+package sgbucket
+
+import (
+	"fmt"
+	"sync"
+)
+
+type VbucketNo uint32
+type VbucketSeq uint64
```

Comment:
Thanks @benbjohnson!  Those are great points.  

@adamcfraser I'm going to back this change out for now since it sounds like it could do more harm than good.